Classes/Views/AttributedStringView.swift

```diff
@@ -8,10 +8,11 @@
 
 import UIKit
 
-protocol AttributedStringViewDelegate: class {
-    func didTapURL(view: AttributedStringView, url: URL)
-    func didTapUsername(view: AttributedStringView, username: String)
-    func didTapEmail(view: AttributedStringView, email: String)
+@objc protocol AttributedStringViewDelegate: class {
+    @objc optional func didTapURL(view: AttributedStringView, url: URL)
+    @objc optional func didTapUsername(view: AttributedStringView, username: String)
+    @objc optional func didTapEmail(view: AttributedStringView, email: String)
+    @objc optional func didTapLabel(view: AttributedStringView, label: String)
```

Comment:
eep, why add `optional`? I guess w/ Swift the compiler forces you to put the `?`, but now if a new delegate forgets to implement a method, the compiler wont catch that bug. Can we leave these required?

Reply:
I actually added it for that exact reason, as we add more detection for additional types we don't necessarily want to enforce every listener to handle it (slash end up with a whole bunch of empty implementations). I can remove it yea, but was intentional!